docs/ingestion/index.md

```diff
@@ -680,10 +680,45 @@ See [Flatten spec](./data-formats.md#flattenspec) for more details.
 ## `ioConfig`
 
 The `ioConfig` influences how data is read from a source system, such as Apache Kafka, Amazon S3, a mounted
-filesystem, or any other supported source system. The `inputFormat` property applies to all
-[ingestion method](#ingestion-methods) except for Hadoop ingestion. The Hadoop ingestion still
-uses the [`parser`](#parser-deprecated) in the legacy `dataSchema`.
-The rest of `ioConfig` is specific to each individual ingestion method.
+filesystem, or any other supported source system.
+
+> The `inputFormat` property applies to all [ingestion methods](#ingestion-methods) except for Hadoop-based ingestion,
+> which uses the deprecated [`parser`](#parser-deprecated) technique inside the legacy `dataSchema` to determine how to
+> read and process source data rows.
+
+The `ioConfig` section contains configuration options that are specific to each individual ingestion method.  For more details, see the documentation provided by each [ingestion method](#ingestion-methods).
+
+| configuration | [`index_parallel`](https://druid.apache.org/docs/latest/ingestion/native-batch.html#ioconfig) | [`kafka`](https://druid.apache.org/docs/latest/development/extensions-core/kafka-ingestion.html#kafkasupervisorioconfig) | [`kinesis`](https://druid.apache.org/docs/latest/development/extensions-core/kinesis-ingestion.html#kinesissupervisorioconfig) | [`hadoop`](https://druid.apache.org/docs/latest/ingestion/hadoop.html#ioconfig) | [`compact`](https://druid.apache.org/docs/latest/ingestion/data-management.html#compaction-ioconfig) |
```

Comment:
This table isn't doing a lot of work b/c there isn't much overlap could we list the properties according to type?